en/config/airspeed.md

```diff
@@ -39,7 +34,7 @@ To calibrate the airspeed sensor:
    If they are swapped then the sensor will read a large negative differential pressure when you blow into the tube, and the calibration will abort with an error. 
    :::
 
-1. Wait for 2-3 seconds before removing the covering (calibration completes silently after several seconds)
+1. The GUI inside *QGroundControl* tells you if the calibration was successful or not. 
```

Comment:
```suggestion
1. The GUI of *QGroundControl* tells you if the calibration was successful or not. 
```

Reply:
Adapted in 9917fb34d4e30166ae4dac815dc01e5380115ae0